sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -552,10 +552,12 @@ private ProxyOptions getDefaultProxyConfiguration(Configuration configuration) {
             return ProxyOptions.SYSTEM_DEFAULTS;
         }
 
-        return getProxyOptions(authentication, proxyAddress);
+        return getProxyOptions(authentication, proxyAddress, configuration,
+            Boolean.parseBoolean(configuration.get("java.net.useSystemProxies")));
     }
 
-    private ProxyOptions getProxyOptions(ProxyAuthenticationType authentication, String proxyAddress) {
+    private ProxyOptions getProxyOptions(ProxyAuthenticationType authentication, String proxyAddress,
```

Comment:
Maybe this is out of scope for this PR but we could consolidate how messaging services support proxies by having this method as a helper in core-amqp (similar to CoreUtils, we can have AmqpUtils). Both EventHubClientBuilder and ServiceBusClientBuilder can just call into this method.